database/index.js

```diff
@@ -0,0 +1,42 @@
+const Sequelize = require('sequelize');
+
+const sequelize = new Sequelize('analystRatings', 'root', 'plantlife', {
+  host: 'localhost',
+  dialect: 'mysql'
+})
+const seedData = require('./seed.js')
+
+const Rating = sequelize.define('rating', {
+  id: {
+    type: Sequelize.NUMBER,
+    autoIncrement: true,
+    primaryKey: true,
+  },
+  ticker: {
+    type: Sequelize.STRING
+  },
+  companyName: {
+    type: Sequelize.STRING
+  },
+  ratingCountBuys: {
+    type: Sequelize.STRING
+  },
+  ratingCountHolds: {
+    type: Sequelize.STRING
+  },
+  ratingCountSells: {
+    type: Sequelize.STRING
+  },
+}, {
+  timestamps: false
+}
+);
+
+Rating.bulkCreate(seedData.sampleRatings).then((ratings) => {
+  console.log("All analyst ratings: ", JSON.stringify(ratings, null, 4))
```

Comment:
Would you need to JSON.stringify the ratings if the initial seedData.sampleRatings input is already in JSON format? In my case I did not use bulkCreate but created in a for-loop, and I did not have to stringify the results(ratings in your case) after a callback from findAll.